src/Html/HtmlGenerator.fs

```diff
@@ -173,19 +173,16 @@ module internal HtmlGenerator =
 
         let htmlType = bindingContext.ProvidedTypeDefinition(asm, ns, typeName, Some typeof<HtmlDocument>, hideObjectMethods = true, nonNullable = true)
         
-        let containerTypes = Dictionary<string, ProvidedTypeDefinition>()
+        let containerTypes = System.Collections.Concurrent.ConcurrentDictionary<string, ProvidedTypeDefinition>()
```

Comment:
Unlike the watchers and the global caches, this dictionary is not ever accessed from multiple threads, so we don't need the concurrentDictionary here, this is a locally scoped item